poc_config/openmrs/apps/common/formLayout.json~

```diff
@@ -0,0 +1,468 @@
+[
```

Comment:
I think maybe this is a temporary file, since it has a `~` at the end of the filename. Do we need this file, or can it be remove?